### PR TITLE
Switch to plyvel as it has better support.

### PR DESCRIPTION
**Pull request by domoritz**
https://plyvel.readthedocs.io

Unfortunately, this needs leveldb installed separately so we want to be careful about this.

### CODE REVIEW

File: search/make_catlas_minhashes.py [hunk 1]
```diff
@@ -189,17 +189,16 @@ def main():
         print('per --no-minhashes, NOT building minhashes database.')
     else:
         path = os.path.join(args.catlas_prefix, 'minhashes.db')
-        db = leveldb.LevelDB(path)
+        db = plyvel.DB(path, create_if_missing=True, error_if_exists=True)
```
Comment (domoritz) [inline search/make_catlas_minhashes.py]: Not sure whether you want to crash if the file already exists.

Comment (ctb) [inline search/make_catlas_minhashes.py]: => unlink file if it already exists.

**Comment by ctb** [comment]
I'd like to close this as #wontfix because (as I just verified) it complicates installation on systems where I don't have root :).

**Comment by domoritz** [comment]
Go ahead.

**Pull request closed without merging**